lib/buffer.js

```diff
@@ -70,6 +70,11 @@ function Buffer(arg, encoding) {
 Object.setPrototypeOf(Buffer.prototype, Uint8Array.prototype);
 Object.setPrototypeOf(Buffer, Uint8Array);
 
+if (Symbol.species && Buffer[Symbol.species] === Buffer) {
```

Comment:
Can you expound on why "being staged, but not shipped" requires node to do language feature detection?